db/migrate/20160111112417_remove_unused_indexes.rb

```diff
@@ -0,0 +1,53 @@
+class RemoveUnusedIndexes < ActiveRecord::Migration
+  def up
+    #was put in place for the aggregation engine, that never used it!
+    remove_index :classifications, column: :workflow_version
+    #this will be used soon for HHMI project group classificaton comparison
+    remove_index :classifications, column: :user_group_id
+    #this compound index is never used but the user_id one is
+    remove_index :memberships, column: [ :user_id, :identity ]
+    #not currently used and should only be used with a more restrictive filter
+    # e.g. user_group_id: X and overlap or contains roles..
+    remove_index :access_control_lists, column: :roles
+    #aggregations only make sense in the context of a subejct, workflow
+    # use the compound index instead
+    remove_index :aggregations, column: :subject_id
+    remove_index :subjects, column: :zooniverse_id
+    #index is not used as it's only for sorting with more restrictive filter
+    remove_index :user_project_preferences, column: :updated_at
```

Comment:
Won't this be used now that this is merged? https://github.com/zooniverse/Panoptes-Front-End/pull/2094/files#diff-4e943d3edc8de512994b6a0031c1ed97R82